cloud_governance/policy/aws/cost_explorer_reports.py

```diff
@@ -0,0 +1,131 @@
+import datetime
+import os
+
+import pandas as pd
+
+from cloud_governance.common.clouds.aws.cost_explorer.cost_explorer_operations import CostExplorerOperations
+from cloud_governance.common.clouds.aws.iam.iam_operations import IAMOperations
+from cloud_governance.common.clouds.aws.sts.sts_oprations import STSOperations
+from cloud_governance.common.elasticsearch.elastic_upload import ElasticUpload
+from cloud_governance.common.google_drive.google_drive_operations import GoogleDriveOperations
+from cloud_governance.main.environment_variables import environment_variables
+
+
+class CostExplorerReports:
```

Comment:
I think we should add a dedicate tests for this method class,
also add documentation